### PR TITLE
add video streaming info to readme

### PR DESCRIPTION
**Pull request by iCollin**
### Summary
It was noted that the video streaming setup for the generic HMI was not so clear.

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -360,4 +362,4 @@ If you aren’t satisfied with the build tool and configuration choices, you can
 
 Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
\ No newline at end of file
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
```
Comment (iCollin) [inline README.md:365]: i used the github.com markdown editor and it added a newline after this text, the content is unchanged

File: README.md [hunk 2]
```diff
@@ -59,14 +59,16 @@ After running the build command, you can launch the Generic HMI in a web browser
 chromium-browser generic_hmi/build/index.html
 ```
 
-### PTU with vehicle modem (optional)
+### PTU w/ Vehicle Modem & Video Streaming
 
-In order to get policy table updates using the vehicle modem, some additional setup is required
+The generic_hmi includes an additional component that is required for in browser video streaming or policy table updates using the vehicle modem.
```
Comment (jacobkeeler) [inline README.md:64]:
```suggestion
The generic_hmi includes an additional backend component that is required for some features, such as in-browser video streaming and policy table updates using the vehicle modem.
```

File: README.md [hunk 3]
```diff
@@ -59,14 +59,16 @@ After running the build command, you can launch the Generic HMI in a web browser
 chromium-browser generic_hmi/build/index.html
 ```
 
-### PTU with vehicle modem (optional)
+### PTU w/ Vehicle Modem & Video Streaming
```
Comment (jacobkeeler) [inline README.md:62]:
```suggestion
### HMI Backend
```

**Review by jacobkeeler — CHANGES_REQUESTED** [review]
Some wording suggestions

**Review by jacobkeeler — APPROVED** [review]

**Pull request merged**